helm/cluster-azure/values.yaml

```diff
@@ -33,6 +33,9 @@ controlPlane:
     usernameClaim: ""
   replicas: 3
   rootVolumeSizeGB: 50
+global:
+  podSecurityStandards:
+    enforced: false
```

Comment:
PSPs will be enabled by default. The steps will be like
- Bump to this version
- Disable PSPs by changing this value
- Bump k8s version

Reply:
Does the above means that (especially interested in the last part here):
- cluster is bumped to the new version of `cluster-azure`
- then `.global.podSecurityStandards.enforced` is change to `true`
- and finally k8s version is bumped with changing the `.internal.kubernetesVersion` property while `cluster-azure` version remains the same?

Reply:
yes